core/css/header.scss

```
@@ -172,7 +172,20 @@
 	/* Right header standard */
 	.header-right {
 		> .header-menu:not(.user-menu) {
-			filter: var(--background-image-invert-if-bright);
+			// For general
+			> .header-menu__trigger {
```

```suggestion
			> button {
```

this should work for all I think

---

I fear the assistant button is not a button but a link...

---

Also I would rather not risk using the button tag here...

---

Then we should probably adjust the two other that do not use `.header-menu__trigger`.
But for the moment your solution works fine!

---

> Also I would rather not risk using the button tag here...

```css
button,
a {}
```

?

---

Even more risky...

---

or even better only apply to `img` tag. Everything else should handle it correctly and we prevent future issues